_projects/decrease_docker_image_size.md

```diff
@@ -0,0 +1,76 @@
+---
+collaborating_projects:
+  - docker-coala-base
+desc: "Find creative ways to decrease the coala Docker image size"
+issues:
+  - https://github.com/coala/docker-coala-base/issues/55
+difficulty: "hard"
+initiatives:
+  - GCI
+markdown: decrease_docker_image_size.md
+mentors:
+  - yukiisbored
+name: "Decrease Docker image size"
```

Comment:
Consider adding a requirements tag so students will know what they should have completed to do this. Alternatively, if you don't think they need reqs, ignore this message.

Reply:
@jayvdb do you think it's a good idea to make them finish CI3 first?

Reply:
Yes.

Reply:
Also, CI1-3 are broken at the moment, as mentioned on Zulip, and nobody has responded to that, so I'd appreciate your help getting that working, otherwise we're f'kd in 2 weeks.